docs/jwst/background_subtraction/main.rst

```diff
@@ -0,0 +1,183 @@
+| Dedicated background     | Default       | Optional          |                             |
++--------------------------+---------------+-------------------+-----------------------------+
+| Nodded point source      | Default       | Optional          |                             |
++--------------------------+---------------+-------------------+-----------------------------+
+| User supplied            |               | Default           |                             |
++--------------------------+---------------+-------------------+-----------------------------+
+| **MIRI MRS:**            |               |                   |                             |
++--------------------------+---------------+-------------------+-----------------------------+
+| Dedicated background     | Default       | Optional          |                             |
++--------------------------+---------------+-------------------+-----------------------------+
+| Nodded point source      | Default       | Optional          |                             |
++--------------------------+---------------+-------------------+-----------------------------+
+| User supplied            |               | Default           |                             |
++--------------------------+---------------+-------------------+-----------------------------+
+
+These background subtraction methods are only available for the observing modes
+listed in the table. Other spectroscopic modes, including NIRCam and NIRISS Wide Field
+Slitless Spectroscopy (WFSS), NIRCam Time Series Grism, NIRISS Single Object Slitless
+Spectroscopy (SOSS), and MIRI LRS slitless, use other ways of handling background.
+
+Image-from-Image Subtraction
+----------------------------
+As explained in the documentation for the :ref:`background <background_step>` step,
+this process combines one or more exposures to be treated as backgrounds into a
+sigma-clipped mean background image, which is then directly subtracted, in
+detector space, from an exposure being processed in the :ref:`calwebb_image2 <calwebb_image2>`
+or :ref:`calwebb_spec2 <calwebb_spec2>` pipelines for imaging or spectroscopic
+data, respectively. For imaging mode observations this is only possible when
+observations of a designated background target have been obtained. For spectroscopic
+modes this is possible either through observations of a designated background target
+or when nodded exposures of a point-like target are obtained (e.g. using the MIRI LRS
+"ALONG-SLIT-NOD" dither pattern for an LRS fixed slit observation). Exposures from
+one nod position can be used as background for exposures at the other nod position,
+assuming the source is point-like.
+
+In either instance, the exposures to be used as background are included in the
+`image2` or `spec2` ASN file used to process the science target exposures, where
+the background exposures are labeled with an ASN member type of "background".
```

Comment:
> `image2` or `spec2` association (ASN) file used...

> with an ASN member `exptype` of "background".

or has "exptype" now permanently changed to "type"

Reply:
No, the attribute name in the ASN file is still "exptype" (unfortunately). I just referred to it in the text of the docs as a member type, because I think that wording is less confusing than saying "exposure type", because that can be interpreted as "EXP_TYPE".